helix-front/server/app.ts

```diff
@@ -27,6 +35,17 @@ if (APP_INSIGHTS_CONNECTION_STRING) {
     .start();
 }
 
+if (httpsProxyAgent && process.env.NODE_ENV === 'production') {
+  // NOTES:
+  //
+  // - `defaultClient` property on `appInsights` doesn't exist
+  // until `.start` is called
+  //
+  // - in development on our laptop (as opposed to a server)
+  // we don't need to go through a proxy.
+  appInsights.defaultClient.config.httpsAgent = httpsProxyAgent;
+}
+
 const app = express();
 const server = http.createServer(app);
 
```

Comment:
changes look good to me
you can actually test this by removing  `appInsights.defaultClient.config.httpsAgent = httpsProxyAgent;` out of the `if` block

Using the proxy agent during localhost dev works too
that way you can verify it doesn't crash or anything like that, then you can leave the as you have it in the `if` block